dev/CommonStyles/InkToolbar_themeresources.xaml

```diff
@@ -1016,27 +1253,31 @@
                                     <VisualState.Setters>
                                         <Setter Target="Border.Background" Value="{ThemeResource InkToolbarFlyoutItemBackgroundThemeBrush}" />
                                         <Setter Target="Content.Foreground" Value="{ThemeResource InkToolbarFlyoutItemForegroundThemeBrush}" />
+                                        <Setter Target="CurrentSelectionIcon.Opacity" Value="0" />
                                     </VisualState.Setters>
                                 </VisualState>
 
                                 <VisualState x:Name="PointerOver">
                                     <VisualState.Setters>
                                         <Setter Target="Border.Background" Value="{ThemeResource InkToolbarFlyoutItemHoverBackgroundThemeBrush}" />
                                         <Setter Target="Content.Foreground" Value="{ThemeResource InkToolbarFlyoutItemHoverForegroundThemeBrush}" />
+                                        <Setter Target="CurrentSelectionIcon.Opacity" Value="0" />
                                     </VisualState.Setters>
                                 </VisualState>
 
                                 <VisualState x:Name="Pressed">
                                     <VisualState.Setters>
                                         <Setter Target="Border.Background" Value="{ThemeResource InkToolbarFlyoutItemPressedBackgroundThemeBrush}" />
                                         <Setter Target="Content.Foreground" Value="{ThemeResource InkToolbarFlyoutItemPressedForegroundThemeBrush}" />
+                                        <Setter Target="CurrentSelectionIcon.Opacity" Value="1" />
```

Comment:
I'm not 100% sure what's going on here, but you can't set an property in two different VisualStateGroups. Should these be in the same group then?

Reply:
Hmmm. This is a new knowledge for me. I have no idea about "can't set an property in two different VisualStateGroups".  Let me create an email to make sure we are on the same page.

---
In reply to: [600893246](https://github.com/microsoft/microsoft-ui-xaml/pull/4507#discussion_r600893246) [](ancestors = 600893246)